src/js/main.js

```diff
@@ -0,0 +1,36 @@
+require('uswds');
+
+let wdfnviz = {};
+
+let load = function(onLoadFnc) {
+    // NOTE: Here we use a try/catch block rather than a global "onerror"
+    // handler, because Babel's polyfills strip some of the exception data out.
+    // This method retains access to the exception object.
+    try {
+        onLoadFnc();
+    } catch (err) {
+        // Send exception to Google Analytics.
+        if (window.ga) {
+            window.ga('send', 'exception', {
+                // exDescription will be truncated at 150 bytes
+                exDescription: err.stack,
+                exFatal: true
+            });
+        }
+        throw err;
+    }
+};
+
+wdfnviz.main = function(onLoadFnc) {
+    if (document.readyState !== 'loading') {
+        load(onLoadFnc);
+    } else {
+        document.addEventListener('DOMContentLoaded', function() {
+                load(onLoadFnc);
+            }, false
+        );
+
+    }
+};
+
+module.exports = wdfnviz;
```

Comment:
Could this be an ES6 module with "main" exported directly? And since we're exporting here, maybe a UMD module of the Rollup-side would make sense.

Reply:
I will admit to still being a little fuzzy about the differences between the different module types. But I will change this to a direct export since that seems to be our pattern.  UMD does make sense for this so it is as flexible as possible. I'll look into that.

Reply:
Might be worth looking into "es" format also. This is what corresponds to the "module" attribute of package.json. Here's how Rich Harris (the Rollup guy) set it up on Leaflet: https://github.com/Leaflet/Leaflet/blob/master/build/rollup-config.js#L38-L52